### PR TITLE
Color schemes: Fix persistence schema

### PR DESCRIPTION
**Pull request by sirreal**
#### Changes proposed in this Pull Request

The persisted user settings for selected color scheme were not being reused in state.

This resulted in console warnings:

Screen Shot 2019-08-02 at 12 14 33

This means the persisted state was rejected and not used.

#### Testing instructions

* Use a previously missing theme and verify that no console warnings are produced (must be dev environment).

### CODE REVIEW

File: client/state/preferences/schema.js [hunk 1]
```diff
@@ -62,15 +62,15 @@ export const remoteValuesSchema = {
...
 				'classic-bright',
-				'laser-black',
-				'powder-snow',
+				'contrast',
+				'midnight',
 				'nightfall',
+				'ocean',
+				'powder-snow',
 				'sakura',
+				'sunset',
```
Comment (sirreal) [inline client/state/preferences/schema.js]: I created this list based on the values here:

https://github.com/Automattic/wp-calypso/blob/360ebd2ea5c7613a86250d8b95d2231452d19355/client/blocks/color-scheme-picker/constants.js#L9-L71

I have no idea whether this is the canonical list.

Comment (sixhours) [reply client/state/preferences/schema.js]: This is the canonical list!

**Comment by sixhours** [comment]
Tested this, and I no longer get errors when I switch to one of the new color schemes. Thanks for catching this!

File: client/state/preferences/schema.js (hunk 1 above)
Comment (sirreal) [reply client/state/preferences/schema.js]: I'll add a comment here to indicate that this list and the schema need to be kept in sync.

**Pull request merged**